### PR TITLE
added documentation for ParsedDirichletBCs

### PR DESCRIPTION
**Pull request by asartori86**

### CODE REVIEW

File: include/parsed_dirichlet_bcs.h [hunk 1]
```diff
@@ -44,6 +44,8 @@ using namespace dealii;
  * std::map<types::global_dof_index,double> dirichlet_dofs;
  * ConstraintMatrix constraints;
  * ...
+ * // the following functions apply the boundary conditions for the
+ * // boundary ids 0,1,6
```
Comment (luca-heltai) [inline include/parsed_dirichlet_bcs.h:48]: Uh? 0,1,6? do they have a special meaning?

File: include/parsed_dirichlet_bcs.h [hunk 2]
```diff
@@ -55,6 +57,13 @@ using namespace dealii;
  * parsed_dirichlet.interpolate_boundary_values(dof_handler,quadrature,constraints);
  * ...
  * parsed_dirichlet.interpolate_boundary_values(mapping,dof_handler,quadrature,constraints);
+ * ...
+ *
+ * // in order to apply the boundary conditions also to the
+ * // boundary id 3, where the normal component is set,
```
Comment (luca-heltai) [inline include/parsed_dirichlet_bcs.h:63]: Again, does the boundary id 3 have a special meaning?

**Comment by asartori86** [comment]
Are comments in a example of code. Should I add "with reference to this example?"

**Comment by luca-heltai** [comment]
No, it's not necessary. I could not see the rest of the code in the review, and I thought they were referring to the general class.

:)

**Pull request merged**